src/System.Windows.Forms/src/System/Windows/Forms/TreeNode.cs

```diff
@@ -2284,12 +2299,58 @@ internal unsafe void UpdateImage()
             User32.SendMessageW(tv, (User32.WM)TVM.SETITEMW, IntPtr.Zero, ref item);
         }
 
+        bool IKeyboardToolTip.AllowsChildrenToShowToolTips() => true;
+
+        bool IKeyboardToolTip.AllowsToolTip() => true;
+
+        bool IKeyboardToolTip.CanShowToolTipsNow() => treeView is not null;
+
+        string IKeyboardToolTip.GetCaptionForTool(ToolTip toolTip) => treeView.ShowNodeToolTips ? ToolTipText : treeView.ControlToolTipText;
+
+        Rectangle IKeyboardToolTip.GetNativeScreenRectangle() => BoundsToScreen(Bounds);
```

Comment:
We can use the exist method and we don't need to create one more method.
```suggestion
        Rectangle IKeyboardToolTip.GetNativeScreenRectangle() => treeView.RectangleToScreen(Bounds);
```